src/components/SnackbarClientProvider.tsx

```diff
@@ -0,0 +1,3 @@
+'use client'
+
+export { SnackbarProvider as SnackbarClientProvider } from 'notistack'
```

Comment:
Notistack can't be put in the layout for the app because it is rendered server side. So we have to add this little wrapper for it where we specify the `use client` directive at the top of the file. Honestly, this is the type of thing that is a little annoying about the react server components, but it is getting easier as more third party libraries support RSC natively.